iml-manager-cli/src/display_utils.rs

```diff
@@ -276,6 +277,17 @@ impl IntoTable for Vec<ServerProfile> {
     }
 }
 
+impl IntoTable for Vec<graphql::ServerProfile> {
```

Comment:
Perhaps, having it under graphql is not the best, but at least, by fixing imports, it could be like

```suggestion
impl IntoTable for Vec<ServerProfile> {
```

Reply:
Done